test/util/nightwatch-commands/axeCheck.js

```diff
@@ -29,11 +29,6 @@ export function command(context, config, _callback) {
       done({ err, results });
     });
   }, [context, (config || {}).rules || this.globals.rules || ['section508', 'wcag2a']], response => {
-    if (response.state !== 'success') {
```

Comment:
Same here. Or if it's only a specific browser for which the aXe response is weird, is there a way to pass in that information and only skip this check in that case.

OR, just change this test to `if (response.state && response.state !== 'success')` and add a comment explaining which browsers this is needed for. (Curious if that's documented anywhere, or if it's just what you encountered when running it.)